.mailmap

```diff
@@ -102,3 +102,5 @@ Zev Benjamin <zev@zulip.com> <zev@mit.edu>
 Zixuan James Li <p359101898@gmail.com>
 Zixuan James Li <p359101898@gmail.com> <39874143+PIG208@users.noreply.github.com>
 Zixuan James Li <p359101898@gmail.com> <359101898@qq.com>
+Joseph Ho <josephho678@gmail.com>
+Joseph Ho <josephho678@gmail.com> <62449508+Joelute@users.noreply.github.com>
```

Comment:
FYI, when you find a list in sorted order, other developers will appreciate if you keep it in order when adding new entries.

- #26378